build/VERSION

```diff
@@ -1 +1 @@
-v3.0.0
\ No newline at end of file
+v3.2.0
```

Comment:
I think we should update this file only after all binaries are published to the Content Gateway, no?
Otherwise, I guess users would get notified before the actual binaries are available.

Reply:
My bad, I reverted it. Thanks